src/Uno.UI/UI/Xaml/Media/RadialGradientBrush.iOSmacOS.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Windows.Foundation;
+using Windows.UI.Xaml.Media;
+using CoreAnimation;
+using CoreGraphics;
+using Uno.Extensions;
+
+namespace Microsoft.UI.Xaml.Media
+{
+	partial class RadialGradientBrush
+	{
+		internal override CALayer GetLayer(CGSize size)
+		{
+			var center = Center;
+			var radiusX = RadiusX;
+			var radiusY = RadiusY;
+
+			var radius = (nfloat)(radiusX + radiusY) / 2.0f;
+
+			var transform = RelativeTransform;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'transform'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5417387)